aspnetcore/security/authentication/social/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Facebook, Google, and external provider authentication in ASP.NET Core
+title: External provider authentication in ASP.NET Core for Facebook and Google
```

Comment:
@serpent5  what do you think of this?
```suggestion
title: Facebook, Google, etc., authentication in ASP.NET Core
```

Reply:
I am also curious what Kirk thinks for this.  For my own vote, It feels odd to use "etc." in a title, and I would hope that we could somehow do something better, but it is a puzzle.

Reply:
Feel free to drop etc. But it's more than FB/goog.

Reply:
I don't have much (any?) SEO experience, but would something like this work?:

> Facebook, Google, and other external provider authentication in ASP.NET Core

Reply:
I think Rick has the title that would work best.  Originally, I wasn't sure if I could drop "external" or "provider", while "Google" and "Facebook" are keywords customers are using and important.

Kirk, your suggestion is a good title, but it is 76 characters, so ASP.NET Core would be cut off:
Facebook, Google, and other external provider authentication in ASP.NET Core

The problem was that Google would cut off the title after 57 characters.  So ASP.NET Core was cut off if it was the end and the rest of the title would fit any framework on the planet, hence all the bounce when folks figured out it was not for their framework.  

I knew it was more than Facebook and Google of course, but trying to throw in "such as" or "like" still resulted in a cut off.  So I put the longer (but not complete) list in the description hoping it would be picked up.

Thanks to you both for your help!

Putting ASP.NET Core at the very front I ended risking the main point being cut off, "external authentication" and Google and Facebook were importantly used keywords for folks searching for this topic.

I wasn't sure if I could drop